db/migrate/20180605010014_add_message_id_column_to_comments.rb

```diff
@@ -0,0 +1,5 @@
+class AddMessageIdColumnToComments < ActiveRecord::Migration
```

Comment:
Oops, i think i set this timestamp in the future lol -- fixed and we'll see if it passes now.